### PR TITLE
kernelci.org: update board members list

### PR DESCRIPTION
**Pull request by gctucker**
Update the list of board members now that elections have taken place. The main change is that Don has been elected AB Chair for 2024.

### CODE REVIEW

File: kernelci.org/content/en/docs/org/board.md [hunk 1]
```diff
@@ -17,19 +17,18 @@ year and voted by the board.
 There is also a Project Manager role to help coordinating things with the wider
 team.
 
-As of today, the Advisory Board is composed of the members listed below with
+As of 2024, the Advisory Board is composed of the members listed below with
 their respective roles, email address and IRC nicknames:
 
 * [Chris Paterson](mailto:<chris.paterson2@renesas.com>) (CIP, Treasurer) - `patersonc`
```
Comment (patersonc) [inline kernelci.org/content/en/docs/org/board.md:23]: Just to help clarify what CIP actually is, i.e. not an acronym relating to KernelCI

```suggestion
* [Chris Paterson](mailto:<chris.paterson2@renesas.com>) (Civil Infrastructure Platform project, Treasurer) - `patersonc`
```

**Comment by gctucker** [comment]
@dzickusrh Hello, are you happy with these changes?

The changes have been deployed on staging: https://static.staging.kernelci.org/docs/org/board/

**Comment by dzickusrh** [comment]
@gctucker - Works for me!

**Review by patersonc — APPROVED** [review]

File: kernelci.org/content/en/docs/org/board.md (hunk 1 above)
Comment (patersonc) [reply kernelci.org/content/en/docs/org/board.md:23]: Were you going to make this change?

Comment (gctucker) [reply kernelci.org/content/en/docs/org/board.md]: Yes sorry, looks like I forgot to push the change last week.  It should all be there now.

**Comment by gctucker** [comment]
Thanks for the reviews.  I'll just wait for the staging website to get updated again with the CIP fixup which I forgot to push last week before merging.

**Comment by gctucker** [comment]
Alright got the changes just in time before the 08:00 UTC staging update and it's looking fine afaict.

**Pull request merged**